server/src/main/java/dev/slimevr/autobone/AutoBone.java

```diff
@@ -529,7 +529,7 @@ public AutoBoneResults processFrames(
 					int frameCursor2 = frameCursor + cursorOffset;
 
 					applyConfig(skeleton1.skeletonConfig);
-					skeleton2.skeletonConfig.setConfigs(skeleton1.skeletonConfig);
```

Comment:
Not sure why I even used `setConfigs`, make sure this still works and lets you apply them properly (Or I can do that <3)

Reply:
I’m at school rn, if you’re not busy you can do it :3 
else I’ll do it when I get home

Reply:
Tested it, Autobone works.